airflow-core/src/airflow/ui/src/utils/logs.ts

```diff
@@ -70,5 +72,10 @@ export const parseStreamingLogContent = (
     }
   }
 
-  return data.content;
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (typeof data === "object" && data !== null) {
```

Comment:
```suggestion
  if (typeof data === "object") {
```

I think we could remove the check and the eslint-disable-next-line above.

Reply:
I don't think so. We only want non-null objects.
```
 FAIL  src/utils/logs.test.ts > parseStreamingLogContent > returns empty array for null data
AssertionError: expected [ null ] to deeply equal []

- Expected
+ Received

- []
+ [
+   null,
+ ]
```

Reply:
Thanks that makes sense.